extensions/applicationinsights-analytics-js/src/JavaScriptSDK/Interfaces/IAnalyticsConfig.ts

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { IExceptionConfig } from "@microsoft/applicationinsights-core-js";
+
+/**
+ * Configuration interface specifically for AnalyticsPlugin
+ * This interface defines only the configuration properties that the Analytics plugin uses.
+ */
+export interface IAnalyticsConfig {
+    /**
+     * A session is logged if the user is inactive for this amount of time in milliseconds. Default 30 mins.
+     */
+    sessionRenewalMs?: number;
```

Comment:
Add
- @since tag only if you can identify which version introduced the configuration
- @default if there is a known default value (otherwise don't add it)
- @example if there are existing examples available to make the configuration value more clear to understand